pkg/admin/facade/user.go

```diff
@@ -69,38 +65,25 @@ func (f *UserFacade) SearchPage(
 	})), nil
 }
 
-func (f *UserFacade) Create(identityDef model.IdentityDef, password string) (string, error) {
-	graph, err := f.Interaction.Perform(
-		&interactionintents.IntentAuthenticate{
-			Kind:                     interactionintents.IntentAuthenticateKindSignup,
-			SuppressIDPSessionCookie: true,
-		},
-		&createUserInput{
-			identityDef: identityDef,
-			password:    password,
+func (f *UserFacade) Create(identityDef model.IdentityDef, password string) (userID string, err error) {
+	// NOTE: identityDef is assumed to be a login ID since portal only supports login ID
+	loginIDInput := identityDef.(*model.IdentityDefLoginID)
+	loginIDKeyType := apimodel.LoginIDKeyType(loginIDInput.Key)
```

Comment:
This is incorrect. You cannot simply say key == type. You use `LoginIDConfig.GetKeyConfig`. If not found, you return ```api.NewInvariantViolated(
				"InvalidLoginIDKey",
				"invalid login ID key",
				nil,
			)``` (as in pkg/lib/interaction/nodes/use_identity_login_id.go)